demo/vl-map-search.html

```diff
@@ -24,15 +22,40 @@ <h2>Kaart met adres zoek functionaliteit</h2>
                     &lt;/vl-map&gt;
                 </code>
             </pre>
-        <div class="demo">
-            <vl-map>
-                <vl-map-search></vl-map-search>
-                <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
-                <vl-map-baselayer-grb></vl-map-baselayer-grb>
-                <vl-map-baselayer-grb-ortho></vl-map-baselayer-grb-ortho>
-            </vl-map>
-        </div>
-    </div>
+  <div class="demo">
+    <vl-map id="vl-map-1">
+      <vl-map-search id="vl-map-search-1"></vl-map-search>
+      <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
+      <vl-map-baselayer-grb></vl-map-baselayer-grb>
+      <vl-map-baselayer-grb-ortho></vl-map-baselayer-grb-ortho>
+    </vl-map>
+  </div>
+</div>
+
+<div>
+  <h2>Bind map</h2>
+  <p>Een vl-map-search element en een vl-map element handmatig te koppelen.</p>
+  <pre>
+    <code class="line-numbers language-markup">
+      &lt;vl-map-search id="vl-map-search-3"&gt;&lt;/vl-map-search&gt;
+      &lt;button onclick="document.querySelector('#vl-map-search-3').bindMap(document.querySelector('#vl-map-3'))"&gt;Bind&lt;/button&gt;
+      &lt;vl-map id="vl-map-3"&gt;
+        &lt;vl-map-baselayer-grb-gray&gt;&lt;/vl-map-baselayer-grb-gray&gt;
+        &lt;vl-map-baselayer-grb&gt;&lt;/vl-map-baselayer-grb&gt;
+        &lt;vl-map-baselayer-grb-ortho&gt;&lt;/vl-map-baselayer-grb-ortho&gt;
+      &lt;/vl-map&gt;
+    </code>
+  </pre>
+  <div class="demo">
+    <vl-map-search id="vl-map-search-3"></vl-map-search>
+    <button onclick="document.querySelector('#vl-map-search-3').bindMap(document.querySelector('#vl-map-3'))">Bind</button>
```

Comment:
Ik heb vl-button toegevoegd aan de dev dependencies zodat de look & feel van de demo pagina wat beter is.